publications-2018.bib

```diff
@@ -663,6 +663,30 @@ @PhdThesis{MengyingXiaoPhD2018
   School                   = {Clemson University},
   Year                     = {2018},
   Url                      = {https://tigerprints.clemson.edu/cgi/viewcontent.cgi?article=3177}
+
+
+@article{doi:10.1093/gji/ggy466,
+  author = {Perry-Houts, Jonathan and Karlstrom, Leif},
+  title = {Anisotropic viscosity and time-evolving lithospheric instabilities due to aligned igneous intrusions},
+  journal = {Geophysical Journal International},
+  volume = {},
+  number = {},
+  pages = {ggy466},
+  year = {2018},
+  doi = {10.1093/gji/ggy466}
```

Comment:
This one should get a `note = {Accepted},`. Having this tag would enable us to more easily locate entries that need to be updated sometime in the future. I'd already added a note to this effect in the README file a few days ago.